base/binaryplatforms.jl

```diff
@@ -377,6 +377,8 @@ function os()
         return "macos"
     elseif Sys.isbsd()
```

Comment:
```suggestion
    elseif Sys.isfreebsd()
```
Or swap it with your new branch (otherwise it never can be reached as isbsd returns true on OpenBSD

Reply:
you're right. thanks